includes/discovery/os/ciscowap.inc.php

```diff
@@ -3,3 +3,7 @@
 if (str_contains($sysDescr, 'Cisco Small Business WAP')) {
     $os = 'ciscowap';
 }
+
```

Comment:
I've left it separate as it's sysDescr and sysObjectId. I'm not fussed either way just felt it was easier to read this way.